test/e2e/apps/cronjob.go

```diff
@@ -261,10 +289,7 @@ var _ = SIGDescribe("CronJob", func() {
 
 		ginkgo.By("Ensuring job was deleted")
 		_, err = e2ejob.GetJob(ctx, f.ClientSet, f.Namespace.Name, job.Name)
-		framework.ExpectError(err)
-		if !apierrors.IsNotFound(err) {
-			framework.Failf("Failed to delete %s cronjob in namespace %s", cronJob.Name, f.Namespace.Name)
-		}
+		gomega.Expect(err).To(gomega.MatchError(apierrors.IsNotFound, fmt.Sprintf("Failed to delete %s cronjob in namespace %s", cronJob.Name, f.Namespace.Name)))
```

Comment:
Good find that `gomega.MatchError` can be passed a `func(error) bool` like `apierrors.IsNotFound` - very useful here! :+1: